### ISSUE TITLE
"What is the width of the opening" allows a maximum of 9.99 meter, not enough

### ISSUE THREAD

**Issue by ratti** [Helium314/StreetComplete#628]
The question "What is the width of the opening" allows a maximum of 9.99 meter, which is not enough for industrial entrances, flood protection gates etc.

**How to Reproduce**
Find that question somewhere, try to answer i.e. "18". Higest number I can type is 9.99 , not 10.

**Expected Behavior**
While some validation  is useful, hangars or flood gates should be possible.

**Does it happen in normal StreetComplete?**
It looks like SCEE merged this feature from a SC alpha I can't test.

**Versions affected**
SCEE 58.22
SC: I can see people discussing this in threads related to 59.0-alpha, but I run 58.2 and don't have that feature.

**Comment by mnalis** [comment]
Can you give an example object where that happens (i.e. where you needed to tag more `9.99m`)?

>  SC: I can see people discussing this in threads related to 59.0-alpha, but I run 58.2 and don't have that feature.

Yes, the quest is present only in StreetComplete `v59.0-alpha1` and [newer](https://github.com/streetcomplete/StreetComplete/blob/ad602be749c088b6b71df44887e12be8cb528940/app/src/main/java/de/westnordost/streetcomplete/quests/barrier_opening/AddBarrierOpening.kt#L24-L25).


So it would be best if you @ratti could report the issue there (and only if it won't be resolved there ask for it here). Any code change that can be made upstream in vanilla StreetComplete, makes a life of SCEE maintainers much easier, especially in the long run!


>  While some validation is useful, hangars or flood gates should be possible.

How are those even tagged in OpenStreetMap? Is the quest being asked on them, i.e.:

https://github.com/Helium314/SCEE/blob/a0804440a2f858b5300b9f2d0a8962542d7e729a/app/src/main/java/de/westnordost/streetcomplete/quests/barrier_opening/AddBarrierOpening.kt#L24-L25

**Comment by ratti** [comment]
Hello,

> Can you give an example object where that happens (i.e. where you needed to tag more 9.99m)?

I guess we can skip the part where I search for specific nodes, by just giving some examples where I had that issue. I was mapping in the Hamburg industrial port area, and I got asked that question on…:

- Stormflood barriers going across multiple road lanes
- Entrances to heavy industrial companies
- Airport hangar doors for jumbo jets
- I didn't come across that, but I guess at the ocean coast there even might be “gates“ wider than 100 meters, wild guess.

This is a photo I made some time ago:
<img width="1041" alt="SCR-20240901-mrhb" src="https://github.com/user-attachments/assets/f5564060-a418-4cdb-97aa-9107d49751df">
That's the flood protection barrier for Hamburg Neumühlen, and it's a “small“ one ;-) 

To be honest, I found it a bit irritating to get that question asked with a wheelchair symbol refering to the main entrance of a steel factory. 

[EDIT]: Yes, I'll make a ticket with SC.
